tools/release.sh

```diff
@@ -2,22 +2,16 @@
 
 set -e
 
-if ! [ -d helio ]; then
+if ! [ -f "helio/blaze.sh" ]; then
```

Comment:
maybe add a comment saying it must run from the repo root or, alternatively, cd there no matter where we run from.